packages/shopping/src/repositories/user-credentials.repository.ts

```diff
@@ -0,0 +1,14 @@
+import {DefaultCrudRepository} from '@loopback/repository';
+import {UserCredentials, UserCredentialsRelations} from '../models';
+import {MongoDataSource} from '../datasources';
+import {inject} from '@loopback/core';
+
+export class UserCredentialsRepository extends DefaultCrudRepository<
+  UserCredentials,
+  typeof UserCredentials.prototype.id,
+  UserCredentialsRelations
+> {
+  constructor(@inject('datasources.mongo') dataSource: MongoDataSource) {
```

Comment:
Why MongoDataSource? UserRepository uses juggler.DataSource 

https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/repositories/user.repository.ts#L28

Reply:
I used `lb4 repository` to create this file 🤷‍♂ 

I am fine to use `juggler.DataSource` for consistency 👍 

Later, we can rework all repositories to use the style scaffolded by `lb4 repository`, but I feel that's out of scope of this pull request.